content/languages/cpp/solution-template.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Solution Setup
+sidebar_label: Solution Setup
+---
+
+# C++ Solution Setup
+
+_TODO: add details_
+
+
+## Solution template
+
+```cpp
```

Comment:
To make it easier to test locally, the solution template can be changed to the following that can be reused:
```cpp
// main.cpp
#include <igloo/igloo_alt.h>
#include <igloo/CodewarsTestListener.h>
using namespace igloo;

// Use `#include` to "copy" contents here
#include "preloaded.h" // use empty file if there's no preloaded
#include "solution.cpp"
#include "tests.cpp"

int main(int, const char *[]) {
  NullTestResultsOutput output;
  TestRunner runner(output);
  CodewarsTestListener listener;
  runner.AddListener(&listener);
  runner.Run();
}
```